### DISCUSSION TITLE
Projects (beta): Faster drag & scroll

### DISCUSSION THREAD

**Discussion by willhennessy** [community/community#5944] [Projects and Issues]
Use case:  
1. I'm managing a list of issues in Table view
2. I see an issue at the bottom of the list (~position 20) that I think should be a high priority
3. I drag the issue up to the top of the list (~position 1)

Problem:  the drag & scroll is so slow that it takes 20-30 seconds to reach the top of the list!  Also it doesn't always scroll right away, so I have to move my cursor around to find the sweet spot to initiate scrolling.

**Comment by willhennessy** [comment]
Hi team, love your work but this issue is still so painful for me! Please help :)

**Comment by dtugend** [comment]
At least on firefox the scrolling and drag and drop is horrible.

I am not sure what they use, but if they dropped the fixed header or used a normal scroll bar for the page that would probably be easier, faster and more reliable, the current frame-like method with strange custom(?) scrollbars is very bad.

I'd rather scroll up to find the table header instead of this and I think with CSS you can make the header sticky with normal scroll bars.

**Comment by kimchi-jjigae** [comment]
Please fix this ASAP, it's really making us consider moving to Jira :(
